docs/tractus-x-release.md

```diff
@@ -0,0 +1,21 @@
+# Tractus-X Release
+
+## Artifacts
+
+- Umbrella Helm Chart(s)
+- Changelog (overarching feat)
+
+## Versioning
+
+- Semantic Versioning
```

Comment:
Calendar Versioning

Reply:
Fixed in [5afb19c](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/5afb19ccc4b03b3b3bfd37bd7ea5f929b3c5f273)